notification-bot/src/index.ts

```diff
@@ -0,0 +1,215 @@
+import { StreamClient } from "@apibara/protocol";
+import {
+  Filter,
+  StarkNetCursor,
+  v1alpha2,
+  FieldElement,
+  EventFilter,
+  FilterBuilder,
+} from "@apibara/starknet";
+
+import { RpcProvider, constants, number, provider, uint256 } from "starknet";
+import { formatUnits } from "ethers";
+import * as dotenv from "dotenv";
+import { BlockNumber } from "starknet";
+import * as fs from "fs";
+import * as toml from "toml";
+import { Logger } from 'logger';
+dotenv.config();
+
+const configPath = process.env.CONFIG_PATH || 'config.toml';
+const config = toml.parse(fs.readFileSync(configPath, 'utf-8'));
+
+const BOT_API_KEY = config.telegram?.bot_api_key;
+const CHATID = config.telegram?.chat_id;
+const CLIENT_URL = config.apibara?.url;
+const CLIENT_TOKEN = config.apibara?.token;
+const GOV_CONTRACT_ADDRESS = config.governance?.gov_contract_address;
```

Comment:
If any of these are not able to be read on startup, the program should exit with an error immediately, these shouldn't be allowed to be null as they're critical to 

If I make a typo in [governance] in config.toml (e.g. [govenrance]) , the program will seem to be running, but do nothing.

Reply:
A check like `if(!BOT_API_KEY || !CHATID || !CLIENT_URL || !CLIENT_TOKEN || !GOV_CONTRACT_ADDRESS) {console.error('required fields in configuration file not found'); exit}`

is sufficient

Reply:
done